client/flutter/lib/pages/home_page.dart

```diff
@@ -116,7 +118,9 @@ class HomePage extends StatelessWidget {
                         itemId: null,
                         method: 'Website link');
                     Share.share(
-                        S.of(context).commonWhoAppShareIconButtonDescription);
+                        S.of(context).commonWhoAppShareIconButtonDescription,
+                        sharePositionOrigin: Rect.fromLTWH(0, 0, size.width, size.height / 2),
```

Comment:
This isn't necessarily a blocker for this since this does work to fix the issue, but I'd suggest that we use the position of the Share button bar instead of placing this randomly on the screen... it feels a little weird that the tooltip is pointing at nothing on the screen. Some example code for that is [here](https://alligator.io/flutter/share-plugin/#adding-the-share-functionality). I'm fine with merging this as-is and opening a separate ticket to track that, or including that change in this PR